src/api/article/author/controller.js

```diff
@@ -0,0 +1,39 @@
+import pick from 'lodash/pick';
+
+import { sendJson } from 'utils/api';
+
+import User, { serializeAuthor } from 'api/user/model';
+
+export const getAll = (req, res, next) =>
+  User.find({ active: true, role: 'author' })
+    .then(users => users.map(serializeAuthor))
+    .then(sendJson(res))
+    .catch(next);
+
+export const create = async ({ body }, res, next) => {
+  const emailRgxp = /^generated-author-\d+@wir.by$/;
+  const capturingRgxp = /^generated-author-(.*?)@wir.by$/;
+
+  let nextNum = 0;
```

Comment:
why not use here [`count`](http://mongoosejs.com/docs/api.html#count_count): `User.find({ email: { $regex: emailRgxp } }).count() + 1`?